app/views/accounts/edit.html.erb

```diff
@@ -54,7 +54,7 @@
         <%= translation_button(:account_name) %>
 
         <label class="flex align-center gap flex-item-grow">
-          <%= form.text_field :name, class: "input txt-large", autocomplete: "off", placeholder: "Name this account", autofocus: true,
+          <%= form.text_field :name, class: "input txt-large", autocomplete: "off", placeholder: "Name this account",
```

Comment:
Editing the name of your account is a one-time action. Every time I go to this page to send invitations or configure bots, my keyboard pops up because of the autofocus, very annoying

Reply:
Let's keep this as is for now, and focus on the typo in this PR.

Reply:
done https://github.com/basecamp/once-campfire/pull/5/commits/64a20278adc37a2967bed954d1309635a29f4520